pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -793,13 +793,13 @@ impl CustomScan for PdbScan {
                 }
 
                 // SearchResults found a match
-                ExecState::RequiresVisibilityCheck {
+                ExecState::Ctid {
                     ctid,
                     score,
                     doc_address,
                 } => {
                     unsafe {
-                        let slot = match check_visibility(state, ctid, state.scanslot().cast()) {
+                        let slot = match fetch_ctid(state, ctid, state.scanslot().cast()) {
                             // the ctid is visible
                             Some(slot) => {
                                 state.custom_state_mut().heap_tuple_check_count += 1;
```

Comment:
We will probably want separate counters in the `Acceptor` for check-only, as opposed to fetch. One alternative would be to put these two counters inside the `VisibilityChecker`, and then add two more for the check-only case.